build/template-OneBranch-releasebuild-netonly.yaml

```diff
@@ -1,42 +1,41 @@
-
 # Release build for -netonly package should be triggered after the normal release build
-variables:
+parameters:
   BuildPlatform: 'AnyCPU'
   BuildConfiguration: 'Release'
-  ProjectFilePath: '$(Build.SourcesDirectory)\src\client\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj'
-  ProjectFolderPath: '$(Build.SourcesDirectory)\src\client\Microsoft.Identity.Client'
+  MsalClientSemVer: '4.55.0-devopsbuild'
+  ProjectFilePath: 'microsoft-authentication-library-for-dotnet\src\client\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj'
```

Comment:
You can just make this `ProjectFile` and use the `ProjectFolderPath` to get the full path